pre_commit/commands/autoupdate.py

```diff
@@ -57,7 +58,10 @@ def _update_repo(repo_config, store, tags_only):
 
     # See if any of our hooks were deleted with the new commits
     hooks = {hook['id'] for hook in repo_config['hooks']}
-    hooks_missing = hooks - (hooks & set(new_repo.manifest_hooks))
+    try:
+        hooks_missing = hooks - (hooks & set(new_repo.manifest_hooks))
+    except InvalidManifestError as e:
+        raise RepositoryCannotBeUpdatedError(e.args[0])
```

Comment:
it makes a bigger diff but it might make more sense to move this up so it reads like this:

```python
try:
    new_hooks = Repository.create(new_config, store).manifest_hooks
except InvalidManifestError as e:
   raise ...

hooks_missing = hooks - set(new_hooks)
```

(note I also adjusted the set logic, not sure why it was `and`ing with `hooks` before, I think the set logic is equivalent now)